.gitignore

```diff
@@ -18,6 +18,9 @@
 # JS/ TS/ es6 env files
 /*/*.env
```

Comment:
We don't ignore env files in the JS versio, can we add an empty one here too so the devs know what parameters they need (like we did in the JS version)